Animations/Fade/LeftBig.js

```diff
@@ -26,5 +28,5 @@ var FadeLeftBig = {
     'transform': 'translateX(-2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeLeftBig', FadeLeftBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeLeftBig', FadeLeftBig);
```

Comment:
This does not need to have `'FadeLeftBig'` passed into it since we have defined a `className` property in `FadeLeftBig`.